doc/man3/SSL_CONF_cmd.pod

```diff
@@ -211,6 +211,17 @@ that there will be no forward secrecy for the resumed session.
 enables strict mode protocol handling. Equivalent to setting
 B<SSL_CERT_FLAG_TLS_STRICT>.
 
+=item B<-no_anti_replay>
+
+Switches off replay protection. With replay protection on OpenSSL will
+automatically detect if a session ticket has been used more than once, TLSv1.3
+has been neotiated and early data is enabled on the server. A full handshake is
+forced if a session ticket is used a second or subsequent time. This option is
+set by default and is only used by servers. Anti-replay measures are required
```

Comment:
The sentence `This option is set by default...` is confusing to me. Is it relay protection or the option to turn it off that's set by default?

Reply:
Yes, please clarify this.

Reply:
Replay protection is on by default. I clarified the wording.